app/src/main/java/com.example.weatherapp/data/mapper/WeatherMapper.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.weatherapp.data.mapper
+
+import java.util.*
+import java.text.SimpleDateFormat
+import com.example.weatherapp.data.model.*
+import com.example.weatherapp.data.services.response.*
+
+class WeatherMapper {
+
+    private fun transformMain(main: MainResponse) = MainDescription(
+        main.temp,
+        main.feels_like,
+        main.temp_min,
+        main.temp_max,
+        main.pressure,
+        main.sea_level,
+        main.gnrd_level,
+        main.humidity,
+        main.temp_kf
+    )
+
+    private fun transformWeather(weather: MutableList<WeatherDescriptionResponse>): MutableList<WeatherDescription> {
+        val weatherList: MutableList<WeatherDescription> = mutableListOf()
+        weather.forEach {
+            weatherList.add(
+                WeatherDescription(it.id, it.main, it.description, it.icon)
+            )
+        }
+        return weatherList
+    }
+
+    private val apiFormat = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.ENGLISH)
```

Comment:
put this constant "yyyy-MM-dd HH:mm:ss" in a constantsUtils class pls

Reply:
Moved strings of date api pattern and date app pattern into a Constants file.